src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1097,6 +1097,7 @@ class DocBaseViewer extends BaseViewer {
                 pageCount={this.pdfViewer.pagesCount}
                 pageNumber={this.pdfViewer.currentPageNumber}
                 scale={this.pdfViewer.currentScale}
+                thumbnailsSidebarIsOpen={this.thumbnailsSidebar ? this.thumbnailsSidebar.isOpen : undefined}
```

Comment:
It seems like our default value should be `false` rather than `undefined` here?

Reply:
updated!